### PR TITLE
Added details to WG BYON Readme

### PR DESCRIPTION
**Pull request by oindrillac**
Added some additional details to the BYON WG Readme.

cc: @pacospace please let me know if there are other resources we can add to make the WG items more discoverable.

### CODE REVIEW

File: wg-byon-build-pipelines/README.md [hunk 1]
```diff
@@ -26,4 +27,10 @@ The [charter](charter.md) defines the scope and governance of the BYON build pip
 - [Open Community Issues/PRs](https://github.com/open-services-group/community/labels/wg%2Fbyon-build-pipelines)
 <!-- BEGIN CUSTOM CONTENT -->
 
+## Details
```
Comment (pacospace) [inline wg-byon-build-pipelines/README.md:30]: what about References?

**Review by pacospace — COMMENTED** [review]
Great! Thanks @oindrillac, only one comment!

File: wg-byon-build-pipelines/README.md [hunk 2]
```diff
@@ -16,6 +16,7 @@ The [charter](charter.md) defines the scope and governance of the BYON build pip
 
 ## Meetings
 * Regular WG Meeting: [Wednesdays at 10:00 ET (East Time)](https://meet.google.com/qkt-yacp-wzm) (weekly). [Convert to your timezone](http://www.thetimezoneconverter.com/?t=10:00&tz=ET%20%28East%20Time%29).
+    * [Meeting Notes and Agenda](https://docs.google.com/document/d/1i1ET2HHqWry-0NK72iDRCEGEoyO7CPzmwOBjr437CVI/edit?usp=sharing)
```
Comment (tumido) [inline wg-byon-build-pipelines/README.md:19]: This change will get overwriten by any change to `sigs.yaml`. To add a line like this edit `sigs.yaml` here
https://github.com/open-services-group/community/blob/b05d0c14d1b63707d7e56e92bbb1d09eea2ccd44/sigs.yaml#L165-L170

And add a `archive_url` property (example here: https://github.com/open-services-group/community/blob/b05d0c14d1b63707d7e56e92bbb1d09eea2ccd44/sigs.yaml#L76 )

Then execute
https://github.com/open-services-group/community/blob/b05d0c14d1b63707d7e56e92bbb1d09eea2ccd44/Containerfile#L16

This command will regenerate the README.md based on the `sigs.yaml` prescription

**Review by tumido — CHANGES_REQUESTED** [review]
Changes to README above `<!-- BEGIN CUSTOM CONTENT -->` are overriden by generator. Please don't change the content manually there.

File: wg-byon-build-pipelines/README.md (hunk 2 above)
Comment (oindrillac) [reply wg-byon-build-pipelines/README.md]: Thanks @tumido Made the changes :)

File: wg-byon-build-pipelines/README.md (hunk 1 above)
Comment (oindrillac) [reply wg-byon-build-pipelines/README.md]: Changed this. Thanks @pacospace

**Review by schwesig — APPROVED** [review]
/lgtm
/approve

**Pull request merged**